drivers/led_strip/ws2812_spi.c

```diff
@@ -83,16 +86,25 @@ static int ws2812_strip_update_rgb(const struct device *dev,
 	const uint8_t one = cfg->one_frame, zero = cfg->zero_frame;
 	struct spi_buf buf = {
 		.buf = cfg->px_buf,
+#if IS_ENABLED(CONFIG_WS2812_STRIP_SPI_PREAMBLE_SUPPORT)
+		.len = cfg->empty_preamble_frames + (cfg->length * 8 * cfg->num_colors),
```

Comment:
This driver follows the old 80 column coding style (except in one place which should be fixed). Please let's not mix styles and make this patch stick to a line length of 80 columns.

Reply:
Agree, however this line has been reverted in the addition commit for postambles